litmus-portal/frontend/src/pages/EditSchedule/index.tsx

```diff
@@ -0,0 +1,504 @@
+                            testValue={Test.weight}
+                            spacing={false}
+                            icon={false}
+                          />
+                        ))}
+                      </div>
+                    </div>
+                  )}
+                </div>
+                <div className={classes.summaryDiv}>
+                  <div className={classes.innerSumDiv}>
+                    <Typography className={classes.col1}>YAML:</Typography>
+                  </div>
+                  <div className={classes.yamlFlex}>
+                    {weights.length === 0 ? (
+                      <Typography>
+                        {' '}
+                        {t('createWorkflow.verifyCommit.errYaml')}{' '}
+                      </Typography>
+                    ) : (
+                      <Typography>
+                        <b>{yamlStatus}</b>{' '}
+                        {t('createWorkflow.verifyCommit.youCanMoveOn')}
+                      </Typography>
+                    )}
+                    <br />
+                    <ButtonFilled
+                      style={{ width: '20%' }}
+                      onClick={handleEditOpen}
+                    >
+                      {t('createWorkflow.verifyCommit.button.viewYaml')}
+                    </ButtonFilled>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          {/* Cancel and Save Button */}
+          <div className={classes.buttonDiv} aria-label="buttons">
+            <ButtonOutlined
+              onClick={() => {
+                history.push({
+                  pathname: `/workflows/`,
+                  search: `?projectID=${projectID}&projectRole=${userRole}`,
+                });
+              }}
+            >
+              Cancel
+            </ButtonOutlined>
+            <ButtonFilled onClick={() => handleNext()}>
+              Save Changes
+            </ButtonFilled>
+          </div>
+
+          <Modal
+            open={open}
+            onClose={handleEditClose}
+            width="60%"
+            modalActions={
+              <ButtonOutlined
+                onClick={handleEditClose}
+                className={classes.closeBtn}
+              >
+                &#x2715;
+              </ButtonOutlined>
+            }
+          >
+            <YamlEditor content={manifest} filename={workflow.name} readOnly />
+          </Modal>
+
+          {/* Finish Modal */}
+          <div>
+            <Modal
+              data-cy="FinishModal"
+              open={finishModalOpen}
+              onClose={handleFinishModal}
+              width="60%"
+              aria-labelledby="simple-modal-title"
+              aria-describedby="simple-modal-description"
+              modalActions={
+                <div data-cy="GoToWorkflowButton">
+                  <ButtonOutlined onClick={handleFinishModal}>
+                    &#x2715;
```

Comment:
can we use an icon file here or from MUI?

Reply:
utf-8 encoding is preferred for such symbols like close, comma, dot, etc.